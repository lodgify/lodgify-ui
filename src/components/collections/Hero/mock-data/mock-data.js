export const navigationItems = [
  { text: 'Home', href: '/' },
  {
    text: 'All properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
];

export const backgroundImageUrl =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg';
