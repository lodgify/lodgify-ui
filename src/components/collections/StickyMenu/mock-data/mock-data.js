export const menuItems = [
  {
    text: 'availability',
    link: '#availability',
    active: true,
  },
  {
    text: 'rates',
    link: '#rates',
  },
  {
    text: 'reviews',
    link: '#reviews',
  },
  {
    text: 'description',
    link: '#description',
  },
];

export const styles = {
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center',
  marginTop: '500px',
  height: '400px',
  width: '100%',
  backgroundColor: '#fdb2b2',
};
