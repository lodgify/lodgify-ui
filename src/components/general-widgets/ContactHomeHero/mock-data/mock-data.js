export const navigationItems = [
  { text: 'Home', href: '/' },
  {
    text: 'All properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  {
    text: 'LOL',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
];

export const currencyOptions = [
  { text: 'EUR', value: 'EUR', label: 'Euro' },
  { text: 'USD', value: 'USD', label: 'US Dollar' },
  { text: 'JPY', value: 'JPY', label: 'Japanese Yen' },
];

export const languageOptions = [
  { text: 'English', value: 'en' },
  { text: 'German', value: 'de' },
  { text: 'Japanese', value: 'jp' },
  { text: 'Italian', value: 'it' },
];

export const backgroundImageUrl =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg';
