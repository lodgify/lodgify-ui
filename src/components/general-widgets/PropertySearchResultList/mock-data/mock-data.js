export const propertySearchResults = [
  {
    bathsNumber: 2,
    bathsText: 'baths',
    bedroomsNumber: 2,
    bedroomsText: 'bedrooms',
    bedsNumber: 2,
    bedsText: 'beds',
    guestsNumber: 2,
    guestsText: 'guests',
    imageUrl:
      'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    pricePerPeriod: '$28000',
    periodText: 'per month',
    propertyAmenities: ['Pool', 'Wifi', 'Washer', 'Kitchen'],
    propertyName: 'The Cat House',
    propertyType: 'Bed and breakfast',
    propertyUrl: '/',
    ratingNumber: 4.7,
  },
  {
    bathsNumber: 1,
    bathsText: 'baths',
    bedroomsNumber: 2,
    bedroomsText: 'bedrooms',
    bedsNumber: 3,
    bedsText: 'beds',
    guestsNumber: 4,
    guestsText: 'guests',
    imageUrl:
      'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    pricePerPeriod: '$1280',
    propertyAmenities: ['Pool', 'Wifi', 'Washer', 'Kitchen'],
    propertyName: 'Space Hub A-675',
    propertyType: 'Space Station',
    propertyUrl: '/',
    ratingNumber: 4.7,
  },
];
