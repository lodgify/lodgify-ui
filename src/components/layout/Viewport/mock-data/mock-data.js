export const navigationItems = [
  {
    text: 'Home',
  },
  {
    text: 'All properties',
    href: '/all-properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  { text: 'Contact Us', href: '/' },
  { text: 'What to do', href: '/' },
];

export const images = [
  {
    alternativeText: 'Two cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Two cats',
  },
  {
    alternativeText: 'Two more cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Two more cats',
  },
  {
    alternativeText: 'Much cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Much cats',
  },
];

export const currencyOptions = [
  { text: 'EUR', value: 'EUR' },
  { text: 'USD', value: 'USD' },
  { text: 'JPY', value: 'JPY' },
];
export const languageOptions = [
  { text: 'English', value: 'en' },
  { text: 'German', value: 'de' },
  { text: 'Japanese', value: 'jp' },
  { text: 'Italian', value: 'it' },
];
export const footerNavigationItems = [
  { href: '/', text: 'Home' },
  { href: '/contact', text: 'Contact' },
  { href: '/pool', text: 'Pool' },
  { href: '/spa', text: 'Spa' },
  { href: '/tennis-court', text: 'Tennis Court' },
  { href: '/gym', text: 'Gym' },
];
export const socialMediaLinks = [
  { href: 'https://twitter.com/lodgify', iconName: 'twitter' },
  { href: 'https://instagram.com/lodgify', iconName: 'instagram' },
];
