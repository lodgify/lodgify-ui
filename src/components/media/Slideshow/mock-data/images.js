const url =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg';

export const images = [
  {
    alternativeText: 'Two cats',
    url,
    sizes: '100w',
    srcSet: `${url}?w=400&h=400&mode=max 320w,
        ${url}?w=800&h=800&mode=max 768w,
        ${url}?w=1024&h=1000&mode=max 1024w`,
    title: 'Two cats',
  },
  {
    alternativeText: 'Two more cats',
    url,
    sizes: '100w',
    srcSet: `${url}?w=400&h=400&mode=max 320w,
        ${url}?w=800&h=800&mode=max 768w,
        ${url}?w=1024&h=1000&mode=max 1024w`,
    title: 'Two more cats',
  },
  {
    alternativeText: 'Much cats',
    url,
    sizes: '100w',
    srcSet: `${url}?w=400&h=400&mode=max 320w,
        ${url}?w=800&h=800&mode=max 768w,
        ${url}?w=1024&h=1000&mode=max 1024w`,
    title: 'Much cats',
  },
  {
    alternativeText: 'No dogs',
    url,
    sizes: '100w',
    srcSet: `${url}?w=400&h=400&mode=max 320w,
        ${url}?w=800&h=800&mode=max 768w,
        ${url}?w=1024&h=1000&mode=max 1024w`,
    title: 'No dogs',
  },
];
