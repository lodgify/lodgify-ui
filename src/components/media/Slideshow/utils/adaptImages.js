/**
 * Adapts image data to the shape required by `react-image-gallery`.
 * @param  {Object[]} images
 * @return {Object[]}
 */
export const adaptImages = images =>
  images.map(({ alternativeText, url, sizes, srcSet, title }) => ({
    original: url,
    originalAlt: alternativeText,
    originalTitle: title,
    sizes,
    srcSet,
  }));
