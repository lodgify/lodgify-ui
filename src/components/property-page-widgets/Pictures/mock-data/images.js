const imageUrl =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=1024&mode=max';

export const images = [
  { imageUrl, imageUrlAlternativeText: 'Entrance', label: 'Entrance' },
  { imageUrl, imageUrlAlternativeText: 'Kitchen', label: 'Kitchen' },
  {
    imageUrl,
    imageUrlAlternativeText: 'Living room',
    label: 'Living room',
  },
  { imageUrl, imageUrlAlternativeText: 'Studio', label: 'Studio' },
  {
    imageUrl,
    imageUrlAlternativeText: 'Garden room',
    label: 'Garden room',
  },
];
