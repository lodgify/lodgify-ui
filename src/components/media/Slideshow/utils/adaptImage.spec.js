import { images } from '../mock-data/images';

import { adaptImages } from './adaptImages';

describe('adaptImages', () => {
  it('should map the input image data correctly', () => {
    const actual = adaptImages(images);

    expect(actual).toMatchSnapshot();
  });
});
