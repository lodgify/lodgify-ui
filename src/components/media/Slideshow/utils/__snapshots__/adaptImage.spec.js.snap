// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adaptImages should map the input image data correctly 1`] = `
Array [
  Object {
    "original": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg",
    "originalAlt": "Two cats",
    "originalTitle": "Two cats",
    "sizes": "100w",
    "srcSet": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=400&h=400&mode=max 320w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=800&h=800&mode=max 768w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=1024&h=1000&mode=max 1024w",
  },
  Object {
    "original": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg",
    "originalAlt": "Two more cats",
    "originalTitle": "Two more cats",
    "sizes": "100w",
    "srcSet": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=400&h=400&mode=max 320w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=800&h=800&mode=max 768w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=1024&h=1000&mode=max 1024w",
  },
  Object {
    "original": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg",
    "originalAlt": "Much cats",
    "originalTitle": "Much cats",
    "sizes": "100w",
    "srcSet": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=400&h=400&mode=max 320w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=800&h=800&mode=max 768w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=1024&h=1000&mode=max 1024w",
  },
  Object {
    "original": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg",
    "originalAlt": "No dogs",
    "originalTitle": "No dogs",
    "sizes": "100w",
    "srcSet": "https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=400&h=400&mode=max 320w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=800&h=800&mode=max 768w,
        https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg?w=1024&h=1000&mode=max 1024w",
  },
]
`;
